_posts/2021-01-15-docviewer-updates.md

```diff
@@ -0,0 +1,100 @@
+
+Finding and fixing bugs in the software is also important in case of a full review like this. Of course, during the re-rolling of this function, we also encountered many cases where a different solution had to be applied than we originally planned. 🐛
+
+The point is that we tried to look at the possibilities from the user's point of view and we were trying to make this feature user friendly.👭
+
+# Style 🎨
+
+We will release another new option to users, which is overwriting styles from a higher level. It is very important to enable users to shape our solution for their own needs. So far there have been relatively few opportunities for it, but we are changing that now:
+
+You can define you style on your own and you can pass it to the component itself, like:
+
+```javascript
+const useDisabledStyle = makeStyles(() =>
+  createStyles({
+    iconButton: {
+      "&:disabled": {
+        opacity: 0.26,
+      },
+    },
+  })
+);
+
+const disabledStyle = useDisabledStyle();
+```
+
+If you use typescript you can also see the possible classes what you can overwrite:
+
+<p align="center">
+<img src="/img/posts/docviewer/overwrite_styles.png">
+</p>
+
+Since our designer always come up with new solutions and the style is constantly changing, we can never consider it done. 🎡
+
+# New features 🎉
+
+For those who have been using sensenet for a long time, it may not be new, but if you have just joined, you may come across the keywords redaction, highlight and annotation for the first time.
+
+**Redaction** 🔳
+
+Sometimes when you are working with sensitive information 🗝️ in documents you don’t want to share them with everyone, just who has the appropriate privileges. But what if you would like to share the other parts of the document? This is the case when you need to apply redaction.
+With this feature you can hide parts of your doc with a black rectangle. If another user would like to see it, it is only possible if she/he has the permission for it, otherwise the page will be full with black squares. 🙈
+
+<p align="center">
+<img src="/img/posts/docviewer/redaction.gif">
+</p>
+
+**Highlight** ⭐
+
+The purpose of highlighting is to draw attention to important information in a text. Highlighting is effective because it asks the reader to check the important parts first, and then gives an effective way to review other later. 📑
+
+<p align="center">
+<img src="/img/posts/docviewer/highlight.gif">
+</p>
+
+**Annotation** 💬
+
+An annotation is extra information associated with a particular point in a document or other piece of information. It can be a note that includes a comment or explanation or anything what you would like to share with the audience in a designated place in the text. 📋
```

Comment:
```suggestion
Annotation is extra information associated with a particular point in a document. It can be a note that includes a comment, explanation or anything what you would like to share with the audience in a designated place in the text. 📋
```